pyvista/plotting/_plotting.py

```diff
@@ -224,6 +224,10 @@ def _common_arg_parser(
     elif culling in ['f', 'frontface']:
         culling = 'front'
 
+    if show_scalar_bar is None:
+        # use theme unless plotting RGB
+        _default = theme.show_scalar_bar or scalar_bar_args is not None
```

Comment:
This PR raises some edge cases.  What do we have to happen if the user passes `scalar_bar_args={}`?  I would argue this might mean no scalar bar.  Should we check truthiness instead?